packages/compiler/src/experimental/mutators.ts

```diff
@@ -103,6 +112,15 @@ const seen = new CustomKeyMap<[MutableType, Set<Mutator> | Mutator[]], Type>(([t
   return key;
 });
 
+/** @experimental */
```

Comment:
```suggestion
/** 
 * Mutate the type graph with some namespace mutation.
 * **Warning** this will most likely endup mutating the entire TypeGraph as every type relate to namespace in some way or another causing parent navigation which in turn would mutate everything in that namespace. 
 * @experimental 
 */
```

Reply:
is that comment actually acurate? if you didn't mutate teh namespace on the way down you don't need to look up right?